.github/workflows/ci.yml

```diff
@@ -9,6 +9,16 @@ concurrency:
   group: ${{ github.workflow }}-${{ github.actor }}-${{ github.head_ref || github.run_id }}
   cancel-in-progress: true
 jobs:
+  typos-check:
+    name: Spell Check with Typos
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout Actions Repository
+        uses: actions/checkout@v3
+      - name: Check spelling with custom config file
+        uses: crate-ci/typos@v1.16.24
+        with:
+          config: ./typos.toml
```

Comment:
I possible would prefer to put it in hack/verify-typos.sh so people can quickly check themselves locally.

Reply:
There seems to be no documentation right now to guide people on how to self-check locally.